OrbitGl/CrashManager.cpp

```diff
@@ -29,11 +29,13 @@ void CrashManagerImpl::CrashOrbitService(GetCrashRequest_CrashType crash_type) {
   GetCrashResponse response;
   grpc::ClientContext context;
 
-  grpc::Status status =
-      crash_service_->CrashOrbitService(&context, request, &response);
-  if (!status.ok()) {
-    ERROR("Grpc call failed: %s", status.error_message());
-  }
+  std::function<void(grpc::Status)> callback = [](::grpc::Status status) {
+    if (!status.ok()) {
+      ERROR("Grpc call failed: %s", status.error_message());
+    }
+  };
+  crash_service_->experimental_async()->CrashOrbitService(&context, &request,
```

Comment:
why experimental and not just async()?

Reply:
I don't see async() available for the `crash_service_`. Maybe I should change CrashService itself somehow...

Reply:
I guess our version of grpc does not yet have it (or rather it is still experimental). You could use completion queue for async calls, or add a deadline to ClientContext (see CreateContext in ProcessManager for example).

Reply:
discussed offline: experimental_async is fine here. will replace with async when grpc version is updated.

Reply:
which version of grpc do we need? do you know?